### DISCUSSION TITLE
Is it possible to turn off ligatures preservation when I call get_svg_image?

### DISCUSSION THREAD

**Discussion by fallingleaflun** [pymupdf/PyMuPDF#3795] [Q&A]
### Description of the bug

I call the `page.get_svg_image()` to obtain a svg and I find that some "st" and "ff" are recognized as one element in the `<use data-text="">`. I spot that theres is a Text Extraction Flag called "TEXT_PRESERVE_LIGATURES" in the parameters of`get_textpage`. Is it possible to turn of the ligatures preservation when I call `page.get_svg_image()`?

### How to reproduce the bug

Due to privacy problem, I cannot offer the pdf, sorry.

### PyMuPDF version

1.24.9

### Operating system

MacOS

### Python version

3.9

**Comment by fallingleaflun** [comment]
<img width="833" alt="image" src="https://github.com/user-attachments/assets/b312dbdc-8d63-41c0-9e53-dad620b5ae4a">
"separate" is spelled incorrectly.

**Comment by JorjMcKie** [comment]
Sorry, there is no way to do this. The ligatures flags only pertains to text extraction.
